src/app/pages/TokenDashboardPage/hook.ts

```diff
@@ -41,10 +49,9 @@ export const useTokenTransfers = (scope: SearchScope, address: string) => {
     network,
     layer, // This is OK since consensus has been handled separately
     {
-      limit: NUMBER_OF_ITEMS_ON_SEPARATE_PAGE,
-      offset: offset,
+      ...pagination.paramsForQuery,
       rel: address,
-      // type: 'evm.log',
+      type: 'evm.log',
```

Comment:
First commit should be a separate PR.
Also, can we combine useTokenTransfers and useAccountTokenTransfers now?